### PR TITLE
Joshuadargan/notfound

### PR DESCRIPTION
**Pull request by joshuadargan**
Adds a not found page with various useful links.

A non-existent non-admin route
image

Not logged in on a non-existent admin route
image

Logged in on a non-existent admin route
image

### CODE REVIEW

File: src/pages/NotFound/NotFoundPage.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            )}
+            {!!!LocalStorage.getToken() &&
+              history.location.pathname.indexOf("admin") !== -1 && (
+                <Button
+                  variant="contained"
+                  color="primary"
+                  onClick={() => history.push(Routes.Login)}
+                >
+                  Admin Login
+                </Button>
```
Comment (AqSlz) [inline src/pages/NotFound/NotFoundPage.tsx:40]: Giving unauthenticated people a button to go to the admin login is weird. 

Even if guessing where the admin login page is easy.

File: src/shared/LayoutContainer.tsx [hunk 2]
```diff
@@ -44,7 +45,7 @@ export const LayoutContainer = ({
             EDUGATOR
           </Typography>
           <Box sx={{ flexGrow: 1 }} />
-          {history.location.pathname !== Routes.Login.toString() && (
+          {LocalStorage.getToken() && (
```
Comment (AqSlz) [inline src/shared/LayoutContainer.tsx:48]: This reintroduces the bug that when the session token expires, the logout button disappears, and the user is forced to refresh the page.

See one of your previous pull requests where you changed this line of code.

File: src/pages/NotFound/NotFoundPage.tsx (hunk 1 above)
Comment (joshuadargan) [reply src/pages/NotFound/NotFoundPage.tsx:40]: I think it is fine if they are at a non-existent /admin route. The /admin/login route is easy to guess and you can also find it on Github.

**Review by marcgabe15 — APPROVED** [review]
Awesome! LGTM :)

File: src/pages/NotFound/NotFoundPage.tsx (hunk 1 above)
Comment (tanner-ropp) [reply src/pages/NotFound/NotFoundPage.tsx]: I think I am good either way.  I agree that whichever we choose, we can't really hide the admin login page well

**Review by tanner-ropp — APPROVED** [review]

**Pull request merged**